packages/transport-native/src/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { NativeModules, Platform } from 'react-native';
+
+const LINKING_ERROR =
+    `The package 'trezor-transport-native' doesn't seem to be linked. Make sure: \n\n${Platform.select(
+        { ios: "- You have run 'pod install'\n", default: '' },
+    )}- You rebuilt the app after installing the package\n` +
+    `- You are not using Expo managed workflow\n`;
+
+console.log(NativeModules);
```

Comment:
this console log should be removed before merge, right? Would be nice to mark it by TODO or FIXME comment in code, but this PR comment might be sufficient

Reply:
removed, no need for this